entrypoint.sh

```diff
@@ -5,13 +5,18 @@ FOLDER="$1"
 GITHUB_USERNAME="$2"
 GITHUB_REPO="$3"
 USER_EMAIL="$4"
+REPO_USERNAME="$5"
+
+if [ -z "$REPO_USERNAME" ] then
+  REPO_USERNAME= $GITHUB_USERNAME
```

Comment:
As far as I can tell: the space should not be there.

I think that it should be:
`REPO_USERNAME="$GITHUB_USERNAME"`

Do you care to change this? Then I can merge it.

And thanks for the PR!

Reply:
Hey there,

I can remove that space. I was also going to test this today to see if it actually solves the problem before requesting to merge it. Just didn't want to do it outside of business hours for us.